windows/9.3.x/sitecore-identity/build.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Current image is named `sitecore-xp-identity`. Please replace that.

Reply:
Done in ba90585af6836ef44fa6d354d31337fc6d256a6a